src/utils/Vote.js

```diff
@@ -0,0 +1,36 @@
+export const voteUp = (items, id) => {
```

Comment:
Helper functions for when we get to voting items up and down.

Reply:
It's beautiful